lib/beaker/logger.rb

```diff
@@ -112,19 +112,19 @@ def initialize(*args)
       end
 
       dests = args
-      dests << STDOUT unless options[:quiet]
+      dests << $stdout unless options[:quiet]
```

Comment:
switching from using STDOUT to using $stdout allows us to specify where logger's stdout should be written to, prior to execution of beaker